script/cycles.js

```diff
@@ -0,0 +1,332 @@
+        alert(`You have entered a string or not an number.`);
+    }
+}
+
+console.log(`You entered ${positiveNum} positive number(s), ${negativeNum} negative number(s), ${zero} zero(s), ${evenNum} even number(s), ${oddNum} odd number(s).`);
+
+// 4. Зацикли відображення днів тижня таким чином: 
+// «День тижня. Хочеш побачити наступний день? » і так до тих пір, поки користувач натискає OK.
+
+let weekdays = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+let today;
+let next;
+
+for (let i = 0; i < 7; i++) {
+    today = weekdays[i];
+    do {
+        next = confirm(`The day of the week is ${today}. Do you want to see the next day?`);
+    } while (next == false)
+}
+
+// maximum
+
+// 1. Гра «Вгадай число». Запропонуй користувачеві загадати число від 0 до 100 і відгадай його наступним способом: 
+// кожну ітерацію циклу діли діапазон чисел навпіл, записуй результат в N і питай у користувача 
+// «Ваше число> N, <N або == N?». Залежно від того що вказав користувач, зменшуй діапазон. 
+// Початковий діапазон від 0 до 100, поділи навпіл і отримай 50. Якщо користувач вказав, що його число> 50, то зміни діапазон на від 50 до 100. 
+// І так до тих пір, поки користувач не вибере == N (буде корисним почитати про алгоритм: "бінарний пошук").
+
+const NUMBER = +prompt(`Enter the integer from 0 to 100.`);
+let max = 100;
+let min = 0;
+let N;
+let answer1 = false;
+let answer2 = false;
+
+if (Math.floor(NUMBER) == NUMBER && NUMBER > 0 && NUMBER < 100) {
+    do {
+        N = Math.floor((min + max) / 2);
+        answer1 = confirm(`Is your number below ${N}?`);
+        answer1 === true ? max = Math.floor(N) : min = Math.floor(N);
+        answer2 = confirm(`Is your number ${N}?`);
+        if ((answer2 === false) && (NUMBER == N)) {
+            console.log(`You are lier, because your number is ${N}`);
+            break;
+        }
+    } while (NUMBER != N)
+    console.log(`Your number is ${N}`);
+} else {
+    console.log("Read the condition and enter the correct number.");
+}
+
+// 2. Виведи таблицю множення для всіх чисел від 2 до 9. Кожне число необхідно помножити на числа від 1 до 10.
+let multiplication;
+
+for (let i = 2; i < 10; i++) {
+    console.log(`Multiplication of numbers by ${i}.`)
+    for (let j = 1; j <= 10; j++) {
+        multiplication = i * j;
+        console.log(`${i} x ${j} = ${multiplication}`);
+    }
+}
+
+// 3. Запитай дату (день, місяць, рік) і виведи наступну за нею дату. 
+// Враховуй можливість переходу на наступний місяць, рік, а також високосний рік.
+
+let someDate = prompt(`Enter a date in the format: 01.01.2000 `);
+let arrSomeDate = someDate.split(".");
+let someDay = +arrSomeDate[0];
+let someMonth = +arrSomeDate[1];
+let someYear = +arrSomeDate[2];
+if (someDate.length == 10 && arrSomeDate.length == 3 && someMonth <= 12 && someDay <= 31) {
+    if ((someDay < 28) || ((someDay < 30) && (someMonth == 4 || someMonth == 6 || someMonth == 9 || someMonth == 11)) || 
```

Comment:
This condition is a big one :)

In that case, it is ok to write 3-4 if statements, instead of 1 giant